pywbem/_subscription_manager.py

```diff
@@ -1321,6 +1342,9 @@ def _create_filter(self, server_id, source_namespace, query,
             Value for the `Name` property of the filter instance, or `None`.
             Mutually exclusive with the `filter_id` parameter.
 
+          source_namespace (:term:`string` or None):
+            A namespace or None.
+
```

Comment:
Same comment as above on supporting both.

Reply:
After looking at it this seemed to be an easy way to keep the possibility of a requirement for the SourceNamespace property at little cost to us.

Reply:
Ok, then let's keep it.